docs/io/visualization/montecarlo_packet_visualization.ipynb

```diff
@@ -0,0 +1,373 @@
+{
```

Comment:
<p>Add Oxford comma: "subsequent points P1, P2, and so on."</p><p> </p><p>"Here mu represents the direction of packet propagation with respect to the radial line"</p><p> </p><p>Remove let from "let say"</p><p> </p><p>"...we use the sine rule and apply it to the triangle..."</p><p> </p><p>Can you fix the alignment of "alpha = ...", it's off to the side for some reason</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/2115/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>